src/module.c

```diff
@@ -988,20 +988,20 @@ int RMCreateSearchCommand(RedisModuleCtx *ctx, const char *name,
 
   int rc = REDISMODULE_OK;
   char *categories;
-  if (!strcmp(aclCategories, "")) {
-    categories = SEARCH_ACL_CATEGORY;
+
+  if (strncmp(name, "_", 1) == 0) {
```

Comment:
Consider a function/macro to detect internal commands

Reply:
`strncmp` is the detection function 🤔 
A macro is possible but not sure I see a good reason to use one here